inc/cleantalk-settings.php

```diff
@@ -522,6 +522,10 @@ function apbct_settings__set_fileds()
             'title'  => __('Exclusions', 'cleantalk-spam-protect'),
             'section' => 'hidden_section',
             'fields' => array(
+                'exclusions__log_excluded_requests'        => array(
+                    'title'       => __('Log excluded requests', 'cleantalk-spam-protect'),
+                    'description' => __('Turn the option on to log some type of the excluded requests, like comments from approved authors or POST requests without an Email address', 'cleantalk-spam-protect'),
+                ),
```

Comment:
Spelling:
**Turn the option on** to log some **type** of **the excluded requests**, like comments from approved authors or POST requests without an Email address
1. Bad sounding. Options: _Enable the option / Turn on the option_
2. If some, then "type" should be plural.